### PR TITLE
expose disk space metrics for RocksDB database directory

### PR DESCRIPTION
**Pull request by jsteemann**
### Scope & Purpose

Added new metrics for the total and the free disk space for the mount used for the RocksDB database directory:

* `arangodb_rocksdb_free_disk_space`: provides the free disk space for the mount, in bytes
* `arangodb_rocksdb_total_disk_space`: provides the total disk space of the mount, in bytes

Should work on Linux already, may not yet work on MacOS or Windows.
Definitely needs manual testing by someone with the respective OSes available.

Docs PR: https://github.com/arangodb/docs/pull/539

- [ ] :hankey: Bugfix 
- [x] :pizza: New feature 
- [ ] :hammer: Refactoring 
- [x] :book: CHANGELOG entry made
- [x] :muscle: The behavior in this PR was *manually tested*
- [ ] :computer: The behavior change can be verified via automatic tests

#### Backports:

- [ ] No backports required
- [ ] Backports required for: if it works, we can backport it to 3.7.

### Testing & Verification

- [x] This change is a trivial rework / code cleanup without any test coverage.

Needs to be used in k8s operator as well, as filesystems may behave different there.

### CODE REVIEW

**Comment by jsteemann** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/11707/

**Comment by jsteemann** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/11711/

**Comment by jsteemann** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/11713/

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,14 @@
 devel
 -----
 
+* Added new metrics for the total and the free disk space for the mount
+  used for the RocksDB database directory:
+
+  * `arangodb_rocksdb_free_disk_space`: provides the free disk space for
+    the mount, in bytes
+  * `arangodb_rocksdb_free_disk_space`: provides the total disk space of
```
Comment (jsteemann) [inline CHANGELOG:9]:
```suggestion
  * `arangodb_rocksdb_total_disk_space`: provides the total disk space of
```

**Comment by jsteemann** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/11719/

**Review by KVS85 — APPROVED** [review]

**Pull request merged**